os/mm/umm_heap/umm_zalloc.c

```diff
@@ -92,13 +92,13 @@ void *zalloc_at(int heap_index, size_t size)
 	ARCH_GET_RET_ADDRESS
 	ret = mm_zalloc(&BASE_HEAP[heap_index], size, retaddr);
 	if (ret == NULL) {
```

Comment:
Please add checking size for zalloc also, `if (size == 0)`.

Reply:
added.